src/ray/pubsub/publisher.h

```diff
@@ -134,6 +135,8 @@ class Subscriber {
   double last_connection_update_time_ms_;
 };
 
+}  // namespace pub_internal
```

Comment:
If you'd like me to just put these classes into the Publisher's protected members or something lmk